src/openrct2/drawing/Drawing.h

```diff
@@ -694,8 +694,11 @@ void ttf_draw_string(rct_drawpixelinfo* dpi, const_utf8string text, int32_t colo
 // scrolling text
 void scrolling_text_initialise_bitmaps();
 void scrolling_text_invalidate();
+
+class Formatter;
+
 int32_t scrolling_text_setup(
-    struct paint_session* session, rct_string_id stringId, uint16_t scroll, uint16_t scrollingMode, colour_t colour);
+    struct paint_session* session, rct_string_id stringId, Formatter& args, uint16_t scroll, uint16_t scrollingMode, colour_t colour);
```

Comment:
```suggestion
    struct paint_session* session, rct_string_id stringId, Formatter& ft, uint16_t scroll, uint16_t scrollingMode, colour_t colour);
```

It's already called `ft` everywhere, so I think it would be useful keeping it as such